Assets/FPS/Scripts/Gameplay/Nautica/AbstractNauticaAgent.cs

```diff
@@ -264,5 +265,15 @@ public bool EnemyWasHit()
             enemyWasHit = false;
             return saveHitState;
         }
+
+        public void SetAgentHealthScore(float score)
+        {
+            agentHealth = score;
```

Comment:
this is slightly confusing for someone coming in, since they might not know relationship between health and score.  since it's used in `GetAgentHealth` and looks like health, but also used in `SetAgentHealthScore` where it looks like a score